test/unit/test-amp-story-player.js

```diff
@@ -831,6 +832,24 @@ describes.realWin('AmpStoryPlayer', {amp: false}, (env) => {
       );
     });
 
+    it('rewind() callback should eventually rewind story without iframe', async () => {
+      const playerEl = win.document.createElement('amp-story-player');
+      attachPlayerWithStories(playerEl, 3);
+
+      const player = new AmpStoryPlayer(win, playerEl);
+
+      await player.load();
+      await nextTick();
+
+      const sendRequestSpy = env.sandbox.spy(fakeMessaging, 'sendRequest');
+      player.rewind('https://example.com/story0.html');
+
+      await player.go(2);
```

Comment:
Do you need this?

Reply:
Yep, it's to test the behavior when the story is not connected to the DOM. I did mess up the story that I was calling `rewind()` on, though. Just fixed it.